### PR TITLE
Switch to use go.etcd.io/bbolt

### LINKED ISSUE #11: update the boltdb import to bbolt
https://github.com/etcd-io/bbolt

bbolt is supported by coreos.

EDIT:
i see there is a PR for this already !!!

### PR DESCRIPTION
**Pull request by annismckenzie**
Closes #11.

### CODE REVIEW

**Comment by catsby** [comment]
I believe there is some duplication here with #12

File: go.mod [hunk 1]
```diff
@@ -3,8 +3,8 @@ module github.com/hashicorp/raft-boltdb
 go 1.12
 
 require (
-	github.com/boltdb/bolt v1.3.1
 	github.com/hashicorp/go-msgpack v0.5.5
 	github.com/hashicorp/raft v1.1.0
+	go.etcd.io/bbolt v1.3.3
```
[the remark this answers is not in the document]
Comment (annismckenzie) [reply go.mod]: done, sorry for the late reply

**Review by mjmac — APPROVED** [review]
Tested this with my own code, verified that it fixes `go test -race` failures on go 1.15.

**Comment by annismckenzie** [comment]
I can't merge but I'd appreciate if someone did because it's been approved. 🙈

**Comment by mjmac** [comment]
@vrazdalovschi: Sorry to nudge, but... ::nudge:: Can we get this landed, please? The old boltdb repo is archived and unmaintained, and fails when built with the race detector enabled.

**Comment by s-christoff** [comment]
Howdy everyone, 
Thanks so much for all the interest in this PR. 
It's currently being looked into and tested to see if there are any raft log binary compatibility issues when it is added into relevant products, or if it can be seamlessly integrated into our products without significant code changes.
I appreciate your patience while we dig into this.

**Comment by mjmac** [comment]
> 
> 
> Howdy everyone,
> Thanks so much for all the interest in this PR.
> It's currently being looked into and tested to see if there are any raft log binary compatibility issues when it is added into relevant products, or if it can be seamlessly integrated into our products without significant code changes.
> I appreciate your patience while we dig into this.

Hi, any updates?

**Comment by s-christoff** [comment]
Hello again,
I am super sorry for the delay. I am currently working on this in #22. 
I appreciate the patience as we tested that this would not be a breaking change for any of our internal users of this. At this time we have not found any breakage, and are using [this](https://blog.golang.org/using-go-modules) method to allow for imports of the old and new version simultaneously. I will definitely give credit where credit is due (to #12 , and here in #19). 
At this time I'm going to close this PR.

**Pull request closed without merging**